app/src/pages/inside/common/launchFiltersToolbar/allLatestDropdown/allLatestDropdown.jsx

```diff
@@ -102,7 +104,12 @@ export class AllLatestDropdown extends Component {
     const label = currentOption && currentOption.label;
     const shortLabel = currentOption && currentOption.shortLabel;
     return (
-      <div ref={this.nodeRef} className={cx('all-latest-dropdown')}>
+      <div
+        ref={this.nodeRef}
+        className={cx('all-latest-dropdown', {
+          active: this.props.activeFilterId === 'all' || this.props.activeFilterId === 'latest',
```

Comment:
1. Please use constants `ALL`, `LATEST`
2. To improve readability I suggest moving this condition to a class method `isActive` and use it here